src/python.rs

```diff
@@ -103,6 +103,26 @@ impl<'p> Python<'p> {
     ///
     /// If `globals` is `None`, it defaults to Python module `__main__`.
     /// If `locals` is `None`, it defaults to the value of `globals`.
+    ///
+    /// # Example:
+    /// ```
+    /// use pyo3::{types::{IntoPyDict, PyBytes, PyDict}, prelude::*};
+    /// let gil = pyo3::Python::acquire_gil();
+    /// let py = gil.python();
+    /// let locals = [("ret", py.None())].into_py_dict(py);
+    /// py.run(
+    ///     r#"
+    /// import base64
+    /// s = 'Hello Rust!'
+    /// ret = base64.b64encode(s.encode('utf-8'))
+    /// "#,
+    ///    None,
+    ///    Some(locals),
+    /// );
```

Comment:
The Result from `py.run` should be handled.

Reply:
:+1: